exercises/src/main/scala/fpinscala/parallelism/Par.scala

```diff
@@ -3,39 +3,213 @@ package fpinscala.parallelism
 import java.util.concurrent._
 
 object Par {
+  /*
+  Par[A] needs to be a type that can contain a computation of type A and
+  also yield the value when asked for it. It also needs to contain a
+  marker for whether its computation is concurrent or sequential. So one
+  possible implementation is:
+
+  case class Par[A](comp: A, concurrent: Boolean)
```

Comment:
Yes. But but the function signature you're looking for is more likely `() => A`. Also, probably wouldn't wrap it in a case class, since the notion of equality doesn't really apply well to function in Scala.